server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -44,6 +44,17 @@
      * The logic for ID is: XXYYZZAA, where XX is major version, YY is minor version, ZZ is revision, and AA is alpha/beta/rc indicator AA
      * values below 25 are for alpha builder (since 5.0), and above 25 and below 50 are beta builds, and below 99 are RC builds, with 99
      * indicating a release the (internal) format of the id is there so we can easily do after/before checks on the id
+     *
+     * IMPORTANT: Unreleased vs. Released Versions
+     *
+     * All listed versions MUST be released versions, except the last major, the last minor and the last revison. ONLY those are required
+     * as unreleased versions.
+     *
+     * Current list of unreleased versions:
```

Comment:
Not sure about the version number below, seem to be something that can go stale easily

Reply:
This happens after every major or minor release, last time it fell on my watch 9e2c049830756dcca2dd29c9b6230e2b394d212d. The comment will soon be out of date perhaps we should change the version bump procedure to remove the last bug fix of the previous minor?

Reply:
@davidkyle 

I fully agree to that, for now I only want to fix the build. Fixing the root cause would be awesome. It's not logical to 1st add and then remove a version.

I was completely lost debugging this error, so my code documentation hopefully helps the next poor developer like me. Search doesn't work on these errors, so many issues that mention "version" in some way.